src/app/views/communities/view/overview/edit/channels/channels.ts

```diff
@@ -56,26 +56,11 @@ export default class RouteCommunitiesViewEditChannels extends BaseRouteComponent
 	}
 
 	async onClickRemoveChannel(channel: CommunityChannel) {
-		const shouldRemove = await ModalConfirm.show(
-			this.$gettext('All posts made on this channel will be ejected from the community'),
-			this.$gettextInterpolate('Remove "%{ title }" channel?', { title: channel.title })
-		);
-
-		if (!shouldRemove) {
-			return;
-		}
-
-		try {
-			await channel.$remove();
-		} catch (e) {
-			console.error(e);
-			Growls.error('Could not remove channel');
-		}
+		await CommunityRemoveChannelModal.show(this.community, channel);
```

Comment:
I think we should redirect to the new channel when moving posts to the a channel.

Reply:
I think we shouldn't, especially since posts will not show immediately, and they were in the middle of management, and not content creation.